File: student-geolocation-java/src/io/demo/studentgeolocation/student/Student.java
package io.demo.studentgeolocation.student;

import io.demo.studentgeolocation.common.Geolocation;

public class Student {

	private String name;
	private Geolocation position;

	public Student(String name, double latitude, double longitude) {
		this.name = name;
		position = new Geolocation(latitude, longitude);
	}

	public String getName() {
		return name;
	}

	public Geolocation getPosition() {
		return position;
	}

	public void setPosition(Geolocation position) {
		this.position = position;
	}

}

File: student-geolocation-python/student_geolocation/models/student.py
from geolocation import Geolocation


class Student:
    """Represent a Student property"""
    def __init__(self, name, latitude, longitude):
        self.name = name
        self.position = Geolocation(latitude, longitude)

    @property
    def name(self):
        return self.name

    @property
    def position(self):
        return self.position
